src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+
+
+@import "./blocks/reset";
+@import "./blocks/variables";
+@import "./blocks/card";
+@import "./blocks/stars";
```

Comment:
good practice is to leave 1 line at the end of the file (please do this fix in each your file)